client/reducers/reducer.js

```diff
@@ -1,7 +1,8 @@
-// import { immutable, fromJS, toJS} from 'immutable'
+// import { immutable, fromJS, toJS} from a.immutable
 import jump from './lib/jump'
 import moveForward from './lib/moveForward.js'
 import levels from '../levels'
+import * as a from './action'
```

Comment:
is this a good variable name? It's fine is answer is 'yes', but have a think about it